dyn_run.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
I suggest to put `dry-run.sh`, `run.sh` and `hosts-csdev-3` to separate directory or to `tests` to keep root dir clean. Your thoughts?